src/systemctl/sysv-compat.c

```diff
@@ -73,30 +73,40 @@ int parse_shutdown_time_spec(const char *t, usec_t *ret) {
 
                 errno = 0;
                 hour = strtol(t, &e, 10);
```

Comment:
hmm, if you touch the parser, I think we should also drop use of strtol() here, and instead use safe_atoul(), so that the "+" syntax is not permitted. Because the fact that we accepted it before really was an accident, and not documented behaviour (your new syntax relies on that fact, after all)